src/olympia/migrations/976-add-last-content-review-date-and-perm.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `addons_addonapprovalscounter` ADD COLUMN `last_content_review` datetime(6) DEFAULT NULL;
+UPDATE groups SET rules=CONCAT(rules, ',Addons:ContentReview') WHERE name IN ('Staff', 'Senior Add-on Reviewers');
```

Comment:
similar to the comment in `initial.json` - we could take the opportunity to reset `Staff` to known, simplified, set of rules.

Reply:
There are a lot of rules for this group on stage that do not start with `Addons:`. I'm not sure about prod, so I'd prefer to address that in a follow-up issue.

Reply:
Filed https://github.com/mozilla/addons-server/issues/6556